services/kue.js

```diff
@@ -8,18 +8,24 @@ const kue = module.exports.kue = require('kue');
 // Note that unlike what the name createQueue suggests, it currently returns a
 // singleton Queue instance. So you can configure and use only a single Queue
 // object within your node.js process.
-const Queue = module.exports.queue = kue.createQueue({
-  redis: {
-    createClientFactory: () => redis.createClient()
-  }
-});
+let Queue = module.exports.queue;
```

Comment:
Doesn't this equal to `let Queue = undefined`? If so rather set it to `let Queue = null;` and put a comment that the instance is lazily created and assigned.